happy-travel/edo
# Markup by the country where the agency is located

Issue: https://github.com/happy-travel/agent-app-project/issues/1373
Insomnia: Edo -> Admin -> Markups -> Location Markups
Tested through unit tests and insomnia

Test case:
1) Add country location markup through POST endpoint ../admin/location-markups and check validators
2) Start availability search by agency from country which markup was added before
3) Check if markup was applied
